### ISSUE TITLE
why are CityJSON files not zipped too?

### ISSUE THREAD

**Issue by hugoledoux** [opencitymodel/opencitymodel#9]
Also, now it looks like that the GML files are smaller, while they are ~6X larger.

Thanks for building this!

**Comment by agilliland** [comment]
Hi @hugoledoux , thanks for the question.  The principle reason the JSON data is not compressed is to make it easy to utilize the data directly without having to worry about decompressing it first.

The most common example of this would be if you wanted to use the data in AWS and read the data directly into applications running in the cloud.  I actually created a simple example of this using AWS Athena to query the data where it sits on S3 without requiring any downloading ... https://github.com/opencitymodel/opencitymodel/blob/master/examples/Query-OpenCityModel-using-AWS-Athena.md

In the case of the GML files we decided that compression was best because the files are massive otherwise and since it's much more involved to parse the data in GML format it was less likely users would intend to stream it directly into other applications.

Even without the compression the JSON data is still a fairly reasonable size to download.
